src/main/java/use_case/login/LoginDataOutput.java

```diff
@@ -0,0 +1,15 @@
+package use_case.login;
+
+public class LoginDataOutput {
```

Comment:
I thought the convention it to call it [Usecase]OutputData, not [Usecase]DataOutput

Reply:
Is it?

Reply:
Yup, see here: https://github.com/Artem-on-ishche/CACoding/blob/master/src/use_case/login/LoginOutputData.java